modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/runtime/resolver/ParameterGroupArgumentResolver.java

```diff
@@ -76,10 +76,9 @@ public T resolve(OperationContext operationContext)
             T group = type.newInstance();
             for (Field parameterField : parameterFields)
             {
-                Object value = operationContext.getParameter(getAlias(parameterField));
-                if (value != null)
+                if (operationContext.hasParameter(getAlias(parameterField)))
                 {
-                    parameterField.set(group, value);
+                    parameterField.set(group, operationContext.getParameter(getAlias(parameterField)));
```

Comment:
this doesn't seem like the exact same behaviour. If the parameter was not applied, then nothing is set. That means that a field with a default value will no longer be assigned to `null`